draft-ietf-add-dnr.xml

```diff
@@ -316,14 +316,14 @@
             </list></t>
         </section>
 
-        <section title="Service Mode vs ADN Only Mode">
-          <t>ServiceMode (Section 2.4.3 of <xref
-          target="I-D.ietf-dnsop-svcb-https"></xref>) SHOULD be used because
-          the Encrypted DNS options are self-contained and do not require any
-          additional DNS queries. The reader may refer to <xref
-          target="RFC7969"></xref> for an overview of advanced capabilities
-          that are supported by DHCP servers to populate configuration data
-          (e.g., issue DNS queries).</t>
+        <section title="ADN Only Mode">
+          <t>The provisioning mode, in which an ADN, a list of IP addresses,
+          and a set of service parameters of the encrypted DNS resolver are
+          supplied to a host, SHOULD be used because the Encrypted DNS options
+          are self-contained and do not require any additional DNS queries.
+          The reader may refer to <xref target="RFC7969"></xref> for an
+          overview of advanced capabilities that are supported by DHCP servers
+          to populate configuration data (e.g., issue DNS queries).</t>
```

Comment:
This still doesn't say what additional queries might be involved if you only get the ADN.  I think that you want to suggest following DDR. Whether or not there is a fallback to A/AAAA is a very important question here and one not to be taken lightly.  It changes the way that SVCB processing works in important ways and you need to be very crisp about that or people will just make stuff up.

Reply:
> This still doesn't say what additional queries might be involved if you only get the ADN. I think that you want to suggest following DDR.

The para right after the one you quoted says the following: 

```
The supplied
   ADN will be processed by a host following the procedure in Section 5
   of [[I-D.ietf-add-ddr](https://datatracker.ietf.org/doc/html/draft-ietf-add-dnr-09#ref-I-D.ietf-add-ddr)].
```